forte/data/multi_pack.py

```diff
@@ -176,6 +176,168 @@ def get_span_text(self, begin: int, end: int):
             "specific data pack to get text."
         )
 
+    def remove_pack(
+        self, index_of_pack: int, clean_invalid_entries: bool = False
+    ) -> bool:
+        """
+        Remove a data pack at index `index_of_pack` from this multi pack.
+
+        Note that if the data pack to be removed is associated with entries
+        in for example MultiPackLink, such entries will become invalid after
+        the data pack is removed so needs to set clean_invalid_entries=True.
```

Comment:
Trying to make this part a bit fluent and precise:

In a multi pack, the data pack to be removed may be associated with some multi pack entries, such as `MultiPackLink`s that are connected with other packs. These entries will become dangling and invalid, thus need to be removed. One can consider removing these links before calling this function, or set the `clean_invalid_entries` to `True` so that they will be automatically pruned.

Reply:
The comments in the multi_pack.py line 185-187 is now changed as suggested.